src/bot.ts

```diff
@@ -172,7 +169,7 @@ export default class Bot {
       text: i18n.__('Tip Sent', {
         sender: sender.screen_name,
         receiver: receiver.screen_name,
-        amount: amount,
+        amount: amountInEth,
```

Comment:
```
amount: amountInEth,
symbol: this.tokens.ETH.symbol,
```

->

```
amount: amount,
symbol: symbol,
```

Tipping in Wei is sort of a "hidden command".
It will be fun to say "🎁 [sender] sent 1000000000000000 wei to [receiver] 🎉"  

(Only in tweet text of `tip` command. Normalize to ETH in other cases)

Reply:
Agreed ;)